tests/convert_and_merge_composable_tests/tag-sorting.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version='1.0' encoding='utf-8'?>
```

Comment:
You probably want to add multiple out ports to test that it is sorting via outport first and then inport?

Reply:
I added additional output to make this check